static/app/components/onboarding/gettingStartedDoc/types.ts

```diff
@@ -34,6 +34,11 @@ export type SelectedPlatformOptions<
   [key in keyof PlatformOptions]: PlatformOptions[key]['items'][number]['value'];
 };
 
+export enum DocsPageLocation {
+  ISSUES_PAGE = 0,
+  PROFILING_PAGE = 1,
+}
```

Comment:
Why only issues and profiling?
For now I would restrict it to `PROFILING_SIDEBAR` and add new ones if needed.

Reply:
Those are the only 2 pages where it is being used, if it will be added to some other page, this could easily be extended.

I didn't want to make it an optional parameter, since this way is more explicit how it is being used